utils/docker/Dockerfile.centos.7

```diff
@@ -96,6 +96,10 @@ RUN echo "$USER:$PASSWD" | chpasswd
 # Create directory for DAOS backend storage
 RUN mkdir /mnt/daos
 RUN chown daos.daos /mnt/daos
+RUN mkdir /var/run/daos_server
+RUN chown daos.daos /var/run/daos_server
+RUN mkdir /var/run/daos_agent
+RUN chown daos.daos /var/run/daos_agent
```

Comment:
I guess we don't really validate the Dockerfiles' ability to run DAOS at this point in CI.  :-(

Reply:
Correct, although we do ask people to do this in the documentation. In this patch, i have removed this step from the documentation and move it to the docker file.